### ISSUE TITLE
SCIENCE/DATA: заморозить авторские screen-оценки как falsification corpus

### ISSUE THREAD

**Issue by lemone112** [Labpics-Team/lab-colors#225]
## Назначение

Сохранить сотни авторских screen-оценок как воспроизводимый исследовательский артефакт, не выдавая одного наблюдателя за population ground truth.

Корпус нужен для:

- поиска сильных контрпримеров;
- восстановления происхождения гипотез Labpics;
- сравнения frozen legacy models и simple baselines;
- enrichment stimulus set для #232;
- regression/falsification tests;
- проверки data/context pipeline до multi-observer study.

Он не задаёт production threshold, default policy, universal cleanliness label или observer probability.

## Терминология

Raw ответы сохраняют исходные формулировки. Derived schema разделяет только то, что действительно различимо по исходному заданию:

- brownness / warm-dark cue;
- whiteness / near-white contamination;
- murkiness / clarity;
- dullness / vividness;
- semantic clean–dirty;
- preference;
- acceptability;
- desired correction;
- family/role identity;
- task performance/distinguishability.

Если исторический ответ не позволяет разложение, он получает `LegacyCompositeLabel`, а не ретроспективную выдуманную семантику.

## Provenance и context fidelity

Для каждой записи восстановить, насколько возможно:

- foreground/background/alpha;
- final rendered image/pixels;
- theme/role/family/source anchor;
- exact task wording/order/available alternatives;
- physical medium и appearance mode;
- output domain/color-management path;
- display/browser/OS/HDR metadata;
- white/black luminance, primaries/SPD, ambient, viewing distance, если измерялись;
- spatial geometry/size/proximity/glyph/kernel/backdrops;
- exposure/adaptation/transition context;
- timestamp/session;
- raw response/latency/confidence, если были;
- source file/commit/model version;
- observer-domain metadata с privacy controls.

Отсутствующее значение всегда `Unknown`. Его нельзя выводить из RGB, имени роли, screenshot dimensions или текущего кода.

Context schema следует #230/#233/#235/#237. Historical records не получают искусственно повышенную fidelity.

## Data governance

Зафиксировать:

- owner/source/license каждого raw artifact;
- consent/publication status;
- наличие PII или private project data;
- retention/withdrawal policy;
- разрешённый уровень публикации: private raw, derived aggregate, public counterexample;
- hash manifest без раскрытия закрытого содержимого;
- запрет передавать private screenshots/metadata в публичный dataset без разрешения.

Observer identity хранится pseudonymous и отдельно от responses. Public reports не содержат идентифицирующие данные.

## Dataset layers

```text
raw/          immutable source artifacts
normalized/   schema-normalized without semantic invention
features/     reproducibly computed facts/signals
splits/       frozen partitions
models/       immutable model manifests
reports/      generated comparisons
manifests/    hashes/licenses/provenance/context versions
```

Raw никогда не переписывается migration; новая normalization создаёт новую version.

## Splits

- `exploratory-origin` — данные, на которых возникали V1/V2 hypotheses;
- `author-challenge` — заранее замороженные strong counterexamples;
- `author-retest` — blind repeated evaluation тем же observer;
- `instrumentation` — записи для проверки pipeline;
- population holdout здесь отсутствует и создаётся только #232.

Challenge split hash фиксируется до изменения следующей model version.

## Сравниваемые модели

Без переобучения прогнать:

- frozen legacy Dirt/Muddiness V1;
- frozen `WarmDarkInteractionV2` из #231;
- no-model/no-change;
- lightness-only, chroma-only, hue/opponent-only;
- simple interaction baselines;
- verified published appearance facts из #234;
- family-relative facts #238;
- mode/context-specific slices;
- CVD simulations только как labeled simulation outputs, не observer truth.

Изменение формулы создаёт новый immutable model ID. Старые results не перезаписываются.

## Отчёт

Показывать:

- agreement with author, не human accuracy;
- ranking/calibration непрерывных signals, где label это позволяет;
- confusion только для действительно binary tasks;
- blind test–retest reliability;
- worst cases;
- results by mode/output/context fidelity;
- near-neutral, near-white, earthy, warm-dark, chromatic-background slices;
- cases where dirty/brown/murky/disliked/unacceptable/identity-failed расходятся;
- missing-context rates;
- no-change baseline;
- model failure reasons and applicability.

Не публиковать population confidence, если dataset его не содержит.

## Связь с production

- production crate не читает labels и не обучается при build;
- corpus живёт в research/experiments;
- CI использует только frozen, license-permitted counterexample manifests;
- один author label не разрешает projection;
- model admission происходит только через #232;
- research artifact не становится runtime dependency.

## TDD / reproducibility

- raw hash mismatch ломает pipeline;
- Unknown context не заполняется default-ом;
- challenge rows недоступны training step;
- model report воспроизводим одной command;
- same model/data/context versions дают same report class;
- license/privacy gate блокирует accidental publication;
- legacy model mutation ломает frozen characterization;
- generated features содержат model/context IDs.

## Acceptance

- [ ] Raw artifacts immutable и имеют provenance/hash/license/privacy manifest.
- [ ] Missing display/appearance/spatial context не реконструируется выдумкой.
- [ ] Historical composite labels сохранены честно.
- [ ] Exploratory/challenge/retest/instrumentation физически разделены.
- [ ] V1/V2/simple/published baselines запускаются reproducibly.
- [ ] Report говорит `agreement with author`, не population accuracy.
- [ ] Blind author retest проведён для заранее выбранного subset, если доступен.
- [ ] Near-neutral, near-white, earthy, warm-dark и different-background cases представлены.
- [ ] Private/PII/licensing controls проверяются автоматически.
- [ ] Production crates не зависят от dataset.
- [ ] README объясняет отличие #225 от population study #232.

## Зависимости

Связано: #230, #231, #232, #233, #234, #235, #237, #238.

Поставляет stimuli/pipeline tests в #232.

Не блокирует удаление ложной production semantics.

**Comment by lemone112** [comment]
Ревью PR #226: текущий rustdoc `DirtClassV2` утверждает, что цветной фон не классифицируется, но реализация `analyze_dirt_rgb_v2` использует `V_fg/V_bg` для любого ненулевого chromatic background; deprecated `ChromaticBackground` больше никогда не возвращается. Это фактический docs↔code drift и до merge должно быть устранено.

Кроме того, `Dirty` сейчас означает любое строго положительное значение `yellow_fraction × relative_blackening`, включая сколь угодно малый сигнал. В отчётах корпуса нельзя интерпретировать этот enum как наблюдаемую категорию человека; честнее сравнивать непрерывный `dirtiness_potential`, а бинарный класс описывать как `interaction-positive/zero` либо явно закрепить операциональный смысл.

Добавление к acceptance:
- rustdoc и Diataxis-дока одинаково определяют домен chromatic background;
- отчёт отдельно показывает class-vs-author agreement и ранговую способность непрерывного potential;
- near-neutral/opponent-zero boundary входит в обязательный falsification corpus.

**Comment by lemone112** [comment]
Закрыто как not planned в рамках owner-directed tracker reset. Это НЕ означает, что acceptance уже реализован: GitHub Issues больше не используются как внутренний roadmap/handoff. Требования прочитаны и перенесены в единый локальный master-roadmap; устойчивый выполненный контракт будет жить только в публичных code/tests/docs и небольших PR. Новый внешний воспроизводимый bug можно открыть отдельным focused Issue.
